### PR TITLE
CEDS-1481 Retrieve Declarations by ID & EORI

### PR DESCRIPTION
**Pull request by ed0906**

### CODE REVIEW

File: app/uk/gov/hmrc/exports/services/DeclarationService.scala [hunk 1]
```diff
@@ -30,4 +30,8 @@ class DeclarationService @Inject()(declarationRepository: DeclarationRepository)
    */
   def save(declaration: ExportsDeclaration): Future[ExportsDeclaration] = declarationRepository.create(declaration)
 
+  def find(eori: String): Future[Seq[ExportsDeclaration]] = declarationRepository.find(eori)
```
Comment (bambuchaAdm) [inline app/uk/gov/hmrc/exports/services/DeclarationService.scala]: I am not sure which declaration would be returned when there is more then one submitted.

Comment (bambuchaAdm) [reply app/uk/gov/hmrc/exports/services/DeclarationService.scala]: Ok there is seq there so is should not be problem.

File: conf/app.routes [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # microservice specific routes
 
 POST        /v2/declaration                                  uk.gov.hmrc.exports.controllers.DeclarationController.post()
+GET         /v2/declaration                                  uk.gov.hmrc.exports.controllers.DeclarationController.get()
```
Comment (PauloPortugal) [inline conf/app.routes]: :+1:

File: test/unit/uk/gov/hmrc/exports/controllers/DeclarationControllerSpec.scala [hunk 3]
```diff
@@ -56,14 +56,14 @@ class DeclarationControllerSpec
     reset(mockAuthConnector, declarationService)
   }
 
-  "POST" should {
+  "POST /" should {
     val post = FakeRequest("POST", "/v2/declaration")
 
-    "return 200" when {
+    "return 201" when {
```
Comment (PauloPortugal) [inline test/unit/uk/gov/hmrc/exports/controllers/DeclarationControllerSpec.scala]: :+1: Awesome!

File: app/uk/gov/hmrc/exports/services/DeclarationService.scala (hunk 1 above)
Comment (ed0906) [reply app/uk/gov/hmrc/exports/services/DeclarationService.scala]: Yeh I eventually see this superseding the "get by eori: endpoint

**Pull request merged**